tce/src/applibs/LLVMBackend/TDGen.cc

```diff
@@ -711,23 +717,21 @@ TDGen::write64bitRegisterInfo(std::ostream& o) {
 
     if (regs64bit_.size() < 1) {
         RegInfo reg = { "dummy64", 0 };
-        writeRegisterDef(o, reg, "DRES0", "Rf64", "", RESERVED);
+        writeRegisterDef(o, reg, "DRES0", "R64", "", RESERVED);
         f64regs = "DRES0";
     } else {
-        writeRegisterDef(
-            o, regs64bit_[0], "DRES0", "Ri64", "DIRES0", RESERVED);
+        writeRegisterDef(o, regs64bit_[0], "DRES0", "R64", "DIRES0", RESERVED);
         for (unsigned i = 1; i < regs64bit_.size(); i++) {
             std::string regName = "D" + Conversion::toString(i);
             std::string aliasName = "DI" + Conversion::toString(i);
             f64regs += regName;
             f64regs += ", ";
-            writeRegisterDef(
-                o, regs64bit_[i], regName, "Rf64", aliasName, GPR);
+            writeRegisterDef(o, regs64bit_[i], regName, "R64", aliasName, GPR);
         }
         f64regs += "DRES0";
     }
     o << std::endl
-      << "def F64Regs : RegisterClass<\"TCE\", [f64], 32, (add "
+      << "def R64FPRegs : RegisterClass<\"TCE\", [f64], 32, (add "
```

Comment:
This also. Seems it's the same Kultala pointed out and you just didn't push your latest changes, right?